app/services/file_workflow_service.py

```
@@ -1155,6 +1160,192 @@ def _recursive_scandir(self, path: Path) -> Iterator[os.DirEntry]:
+        For each non-local cold storage location that supports listing managed files,
+        this method pages through all remote files and:
+        - updates last_seen + status for entries already in inventory, and
+        - creates new FileInventory (and FileRecord) entries for files found in the
+          remote backend that are missing from the local inventory.
+        """
+        synced = 0
+        for location in path.storage_locations:
+            try:
+                backend = get_backend(location)
+            except Exception:
+                continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)